### PR TITLE
Enable cancel button for all users

### PR DESCRIPTION
**Pull request by evansd**
This has now been wired up on the job-runner side and tested:
https://github.com/opensafely-core/job-runner/pull/186

### CODE REVIEW

File: jobserver/templates/job_detail.html [hunk 1]
```diff
@@ -158,16 +158,18 @@ <h3 class="text-center">Tools</h3>
...
     </form>
 
+    {% if request.user.is_superuser %}
     <form class="mb-3" method="POST" action="{{ job.get_zombify_url }}">
       {% csrf_token %}
 
       <button class="btn btn-block btn-danger" type="submit">Zombify</button>
     </form>
     {% endif %}
+    {% endif %}
```
Comment (ghickman) [inline jobserver/templates/job_detail.html:172]: We'll want to end the authenticated and authorized user check at the end of the Cancel button, Zombification is only for us superusers still.

**Review by ghickman — CHANGES_REQUESTED** [review]
Just one small thing!

File: jobserver/templates/job_detail.html (hunk 1 above)
Comment (evansd) [reply jobserver/templates/job_detail.html:172]: I think the effect is that same either way (the superuser check is still there, just moved down) but your suggestion makes things clearer.

Comment (ghickman) [reply jobserver/templates/job_detail.html:172]: Oh yes, I see what you mean now 👍

**Review by ghickman — APPROVED** [review]

**Pull request merged**